commit e70e8e91f97496a7df3140dd061b1a48b73d3251 (refs/changes/39/4939/2)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-10-10T19:02:31+00:00 (12 months ago)
    
    TRITON-800 "Bhyve Max Vcpus" should be ignored if "Bhyve Capable" is not true
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 27b790d..c764497 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "assert-plus": "~1.0.0",
     "bunyan": "1.8.5",
     "libuuid": "0.2.1",
-    "joyent-schemas": "git+https://github.com/joyent/schemas.git#9f0791091868f449c8933902b556652e6c24875c",
+    "joyent-schemas": "git+https://github.com/joyent/schemas.git#09cebfca94169601568fec60f7a6144130f7fd31",
     "jsprim": "1.3.1",
     "vasync": "~1.6.4"
   },
